### PR TITLE
Update zh-Hant

### PR DESCRIPTION
**Pull request by open3**
**Details of Changes**
- Updated Chinese Traditional translations

**Validation**
- [x] Built and ran the app
- [x] Tested the changes for accessibility

### CODE REVIEW

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 1]
```diff
@@ -3396,298 +3375,299 @@ Files does not collect, store, share or publish any personal information.
...
         <trans-unit id="ConfictingItemsDialogOptionApplyToAll.Text" translate="yes" xml:space="preserve">
           <source>Apply to all</source>
-          <target state="new">Apply to all</target>
+          <target state="translated">全部應用</target>
         </trans-unit>
         <trans-unit id="PropertyPartOfSet" translate="yes" xml:space="preserve">
           <source>Part of set</source>
-          <target state="new">Part of set</target>
+          <target state="translated">集數</target>
+          <note from="MultilingualEditor" annotates="source" priority="4"></note>
```
Comment (BanCrash) [inline Files/MultilingualResources/Files.zh-Hant.xlf:3494]: Is this note intentional?

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 2]
```diff
@@ -3396,298 +3375,299 @@ Files does not collect, store, share or publish any personal information.
...
+* 個人資料收集聲明 *
+Files 不會收集、存儲、共享或發布任何使用者的個資。
 
-*Non-personal Information Collection*
+* 非個資數據收集聲明 *
 
-We use App Center to keep track of app usage, find bugs, and fix crashes. All information sent to App Center is anonymous and free of any user or contextual data.
-  </target>
+我們使用 App Center 來追蹤使用者在應用程式的使用情況，並藉此查找錯誤及修復問題。所有數據都是以匿名身分回傳至 App Center，你的個資也不會被回傳。
+</target>
```
Comment (BanCrash) [inline Files/MultilingualResources/Files.zh-Hant.xlf:3385]:
```suggestion
  </target>
```

Just a little change for keeping indentation consistency.

**Review by BanCrash — CHANGES_REQUESTED** [review]
It's almost completed, just let me know if that comment is intentional and fix the little indentation issue and this will be approved ;)

**Comment by open3** [comment]
All the issues have been resolved 👍

**Review by BanCrash — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@open3 thank you!

**Pull request merged**